cmd/k8s-operator/main.go

```diff
@@ -7,14 +7,10 @@
 package main
 
 import (
```

Comment:
can we rename this file to `operator.go` and the test to `operator_test.go`?

Reply:
Done.